src/main/resources/functions/createGroup.xml

```diff
@@ -0,0 +1,75 @@
+        </class>
+        <payloadFactory media-type="json" template-type="freemarker">
+            <format>
+                <![CDATA[{
+                                                                                        <#if (args.arg1)?has_content>
+                                    "description": "${args.arg1}",                                </#if>
+                                                                                            <#if (args.arg2)?has_content>
+                                    "displayName": "${args.arg2}",                                </#if>
+                                                                                            <#if (args.arg3)?has_content>
+                                    "groupTypes": ${args.arg3},                                </#if>
+                                                                                            <#if (args.arg4)?has_content>
+                                    "mailEnabled": "${args.arg4}",                                </#if>
+                                                                                            <#if (args.arg5)?has_content>
+                                    "mailNickname": "${args.arg5}",                                </#if>
+                                                                                            <#if (args.arg6)?has_content>
+                                    "securityEnabled": "${args.arg6}",                                </#if>
+                                                                                            <#if (args.arg7)?has_content>
+                                    "visibility": "${args.arg7}"                                </#if>
+                                                                                        }]]>
+            </format>
+            <args>
+                <arg evaluator="xml" expression="$func:description"/>
+                <arg evaluator="xml" expression="$func:displayName"/>
+                <arg evaluator="xml" expression="$func:groupTypes"/>
+                <arg evaluator="xml" expression="$func:mailEnabled"/>
+                <arg evaluator="xml" expression="$func:mailNickname"/>
+                <arg evaluator="xml" expression="$func:securityEnabled"/>
+                <arg evaluator="xml" expression="$func:visibility"/>
+            </args>
+        </payloadFactory>
+        <property name="DISABLE_CHUNKING" scope="axis2" type="STRING" value="true"/>
+        <property name="messageType" value="application/json" scope="axis2"/>
+        <property name="ContentType" value="application/json" scope="axis2"/>
+        <header name="Authorization" expression="get-property('_ACCESS_TOKEN_')" scope="transport"/>
+        <call>
+            <endpoint>
+                <http method="POST" uri-template="{uri.var.base}{+uri.var.urlPath}{+uri.var.urlQuery}"/>
+            </endpoint>
+        </call>
```

Comment:
In other connectors, I have seen that we remove unnecessary headers sent by the server. Do we need to do that here as well?

Reply:
784b8a6dcd9050705c71c825080a52bebd6bfa98